docs-md/tutorials/embedded-connect.md

```diff
@@ -0,0 +1,334 @@
+status.storage.replication.factor=1
+EOF
+```
+
+4. Start ksqlDB and PostgreSQL
+------------------------------
+
+From a directory containing the ``docker-compose.yml`` file created in the first step, run this command in order to start all services in the correct order:
+
+```bash
+docker-compose up
+```
+
+5. Connect to PostgreSQL
+------------------------
+
+Establish an interactive session with PostgreSQL by running this command:
+
+```bash
+docker exec -it postgres psql -U postgres
+```
+
+6. Populate PostgreSQL with vehicle/driver data
+-----------------------------------------------
+
+From within the session opened in the previous step, run these SQL statements to set up our driver data. We will ultimately join this PostgreSQL data with our event streams in ksqlDB:
+
+```sql
+CREATE TABLE drivers (
+  driver_id integer PRIMARY KEY,
+  make text,
+  model text,
+  year integer,
+  license_plate text,
+  rating float
+);
+
+INSERT INTO drivers (driver_id, make, model, year, license_plate, rating) VALUES
+  (0, 'Toyota', 'Prius',   2019, 'KAFKA-1', 5.00),
+  (1, 'Kia',    'Sorento', 2017, 'STREAMS', 4.89),
+  (2, 'Tesla',  'Model S', 2019, 'CNFLNT',  4.92),
+  (3, 'Toyota', 'Camry',   2018, 'ILVKSQL', 4.85);
+```
+
+7. Start ksqlDB's interactive CLI
+---------------------------------
+
+ksqlDB runs as a server which clients connect to in order to issue queries.
+
+Run this command to connect to the ksqlDB server and enter an interactive command-line interface (CLI) session:
+
+```bash
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+8. Create source connector
+--------------------------
+
+Next, we are going to make our PostgreSQL data accessible to ksqlDB by creating a *source* connector. From within the ksqlDB session opened in the previous step, run this command:
+
+```sql
+CREATE SOURCE CONNECTOR jdbc_source WITH (
+  'connector.class'          = 'io.confluent.connect.jdbc.JdbcSourceConnector',
+  'connection.url'           = 'jdbc:postgresql://postgres:5432/postgres',
+  'connection.user'          = 'postgres',
+  'topic.prefix'             = 'jdbc_',
+  'table.whitelist'          = 'drivers',
+  'mode'                     = 'incrementing',
+  'numeric.mapping'          = 'best_fit',
+  'incrementing.column.name' = 'driver_id',
+  'key'                      = 'driver_id',
+  'value.converter.schemas.enable' = false);
+
+```
+
+Once the source connector is created, it will import any PostgreSQL tables matching the specified ``table.whitelist``. Tables are imported via Kafka topics, one topic per imported table. Once these topics are created, we may interact with them just like any other Kafka topic used by ksqlDB.
+
+9. View imported topic
+----------------------
+
+From within your ksqlDB CLI session, run this command to verify that the drivers table has been imported. Since we’ve specified ``jdbc_`` as our topic prefix, you should see a ``jdbc_drivers`` topic in the output of this command:
```

Comment:
```suggestion
In the ksqlDB CLI session, run the following command to verify that the `drivers` table has been imported. Because you specified `jdbc_` as the topic prefix, you should see a `jdbc_drivers` topic in the output.
```